src/native-ads/NativeAdsManager.ts

```diff
@@ -6,6 +6,7 @@ const { CTKNativeAdManager, CTKNativeAdEmitter } = NativeModules;
 const nativeAdEmitter = new NativeEventEmitter(CTKNativeAdEmitter);
 
 const EVENT_DID_BECOME_VALID = 'AdsManagerDidBecomeValid';
+const EVENT_DID_BECOME_ERROR = 'AdsManagerDidBecomeError';
```

Comment:
Perhaps,`EVENT_DID_BECOME_INVALID` would fit better?

Reply:
Yes, you are right. And I think, that we also should set `this.isValid = false`. How will it affect on `onAdsLoaded()`  function?

Reply:
`onAdsLoaded` should work correctly when `isValid = false` as it will simply return a subscription waiting for the manager to become valid again.